### PR TITLE
🥑 Implement multiple category selection

### PR DESCRIPTION
**Pull request by kenetec**
Implements multiple category selection on front-end and edits resource schema on back-end.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (jackieo5023) [inline yarn.lock]: Remove?

File: client/src/utils/api.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 const axios = require('axios');
 
 const instance = axios.create({
-  baseURL: 'https://ymca.now.sh',
```
Comment (jackieo5023) [inline client/src/utils/api.js]: ^

**Comment by jackieo5023** [comment]
@kenetec and I realized that the current deployment of this branch is a little buggy since the client points to the backend on master, not this branch.  Locally, everything should be functioning as it should, however.

File: client/src/components/ResourceForm.js [hunk 3]
```diff
@@ -70,10 +70,25 @@ const ResourceForm = (props: FormProps) => {
...
+          const { name, category, subcategory, ...result } = resource.result;
+
+          // generate initial category values
+          const categoryInitialValues = [];
+          for (let i = 0; i < category.length; ) {
+            categoryInitialValues.push(
+              category[i] + CAT_SUB_SPLITTER + subcategory[i],
+            );
+
+            i += 1;
```
Comment (n3a9) [inline client/src/components/ResourceForm.js]: Why the split?

Comment (kenetec) [reply client/src/components/ResourceForm.js]: That way we can denote which subcategory belongs to what category.

**Pull request merged**